helm/aws-load-balancer-controller/README.md

```diff
@@ -214,3 +214,5 @@ The default values set by the application itself can be confirmed [here](https:/
 | `enableBackendSecurityGroup`                | If enabled, controller uses shared security group for backend traffic                                    | `true`                                                                             |
 | `backendSecurityGroup`                      | Backend security group to use instead of auto created one if the feature is enabled                      | ``                                                                                 |
 | `disableRestrictedSecurityGroupRules`       | If disabled, controller will not specify port range restriction in the backend security group rules      | `false`                                                                            |
+| `objectSelector.matchExpressions`           | Webhook configuration to select specific pods by specifying the expression to be matched                 | `app.kubernetes.io/name NotIn {{ include "aws-load-balancer-controller.name" . }}` |
```

Comment:
The default should be None in this case. The load balancer controller name is currently hardcoded in the template and not configurable.